ddtrace.gemspec

```diff
@@ -15,6 +15,8 @@ Gem::Specification.new do |spec|
 
   spec.summary     = 'Datadog tracing code for your Ruby applications'
   spec.description = <<-DESC.gsub(/^\s+/, '')
+    NOTICE: This gem has been renamed to `datadog` since 2.0.0. Please use `datadog` instead of `ddtrace`.
```

Comment:
Technically 2.0.0 doesn't exist _yet_ for ddtrace, but up to you if you prefer to leave the text as-is ;)